### ISSUE TITLE
Fatal Error if no or not existing Order-Key

### ISSUE THREAD

**Issue by Snickbrack** [heidelpay/woocommerce-heidelpay#19]
```
Fatal error: Uncaught Error: Call to a member function get_payment_method() on boolean in wp-content/plugins/woocommerce-heidelpay-1.3.0/includes/abstracts/abstract-wc-heidelpay-payment-gateway.php:486
Stack trace:

#0 /wp-includes/class-wp-hook.php(288): WC_Heidelpay_Payment_Gateway->addPayInfo('Vielen Dank. De...')
#1 /wp-includes/plugin.php(203): WP_Hook->apply_filters('Vielen Dank. De...', Array)
#2 /wp-content/plugins/woocommerce/templates/checkout/thankyou.php(83): apply_filters('woocommerce_tha...', 'Vielen Dank. De...', NULL)
#3 /wp-content/plugins/woocommerce/includes/wc-core-functions.php(208): include('/var/www/vhosts...')
#4 /wp-content/plugins/woocommerce/includes/shortcodes/class-wc-shortcode-checkout.php(241): wc_get_template('checkout/thanky...', Array) in /woocommerce-heidelpay-1.3.0/includes/abstracts/abstract-wc-heidelpay-payment-gateway.php on line 486
```

**Comment by Snickbrack** [comment]
```
    public function addPayInfo($orderReceivedText)
    {
        $order = $this->getOrderFromKey();
        if ($order === null || $order->get_payment_method() !== $this->id) { <== $order is not null and $order has no function "get_payment_method"
            return $orderReceivedText;
        }
        $paymentInfo = $order->get_meta('heidelpay-paymentInfo');
        if (!empty($paymentInfo)) {
            $orderReceivedText .= '<p>' . $paymentInfo . '</p>';
        }
        return $orderReceivedText;
    }
```

**Comment by Snickbrack** [comment]
Original Order-Key was "wc_order_5bcb5ee682907" and after I changed it to"wc_order_5bcb5ee672907" I get this error above

**Comment by Ryouzanpaku** [comment]
Solved in version 1.5.0
